pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1920,7 +1921,11 @@ func handleDelete(ctx *domainContext, key string, status *types.DomainStatus) {
 	// inactivation i.e. those preserved across reboots?
 	for _, ds := range status.DiskStatusList {
 		if status.IsContainer {
-			continue
+			containerACIDir := types.VerifiedAppImgDirname + "/" + status.ImageID.String()
```

Comment:
Why do we need to calculate it? Is it better to be calculated once by verifier and stored in Status? Then you can just remove it. This way - the rest of the agents don't really need to know the logic of naming at all..